packages/botonic-plugin-ai-agents/src/index.ts

```diff
@@ -0,0 +1,23 @@
+import { BotContext, Plugin } from '@botonic/core'
+
+import { AiAgentClient } from './ai-agent-client'
+import { AiAgentArgs, AiAgentResponse } from './types'
+
+export default class BotonicPluginAiAgents implements Plugin {
+  public aiAgentClient: AiAgentClient
+
+  constructor() {
+    this.aiAgentClient = new AiAgentClient()
+  }
+
+  pre(): void {
```

Comment:
If this was needed, I think it's because in types it is declared as non optional. Ow, it can be removed

Reply:
It is necessary for it to be considered a plugin, otherwise it cannot be added as a plugin.